src/core/ruby_version.rs

```diff
@@ -919,6 +919,88 @@ macro_rules! get_stack_frame_2_5_0(
     )
 );
 
+macro_rules! get_stack_frame_3_3_0(
+    () => (
+        fn get_stack_frame<T>(
+            iseq_struct: &rb_iseq_struct,
+            cfp: &rb_control_frame_t,
+            source: &T,
+        ) -> Result<StackFrame> where T: ProcessMemory {
+            let mut ep = cfp.ep as *mut usize;
```

Comment:
Here to 935 is copy-pasta from above where it gets the cfunc names, because it did the exact algo i needed (copying ruby/ruby)